app/src/main/AndroidManifest.xml

```diff
@@ -37,7 +37,9 @@
             </intent-filter>
         </activity>
 
-        <activity android:name=".feature.main.view.MainActivity">
+        <activity android:name=".feature.main.view.MainActivity" />
```

Comment:
is all activity will support both portrait and landscape?

Reply:
Will add `android:screenOrientation="portrait"` kak